src/models/Execution/__mocks__/mockTaskExecutionsData.ts

```diff
@@ -51,18 +51,15 @@ export const createMockTaskExecutionsListResponse = (length: number) => {
             const startedAt = dateToTimestamp(
                 new Date(Date.now() - 1000 * 60 * (idx + 1))
             );
-            const phase =
-                idx < length - 1
-                    ? TaskExecutionPhase.FAILED
-                    : TaskExecutionPhase.SUCCEEDED;
+            const phase = random(Object.keys(TaskExecutionPhase).length - 1);
```

Comment:
I don’t think random is a good idea in test data due to it breaking the concept of determinism in tests. Can we rework this so it’s consistently reproducible?

Reply:
I probably should have left a comment. These are only used in stories, so the randomness, is desirable because it provides variation to the data when viewing it.